tools/depends/target/libspdlog/Makefile

```diff
@@ -0,0 +1,67 @@
+-include ../../Makefile.include
+DEPS = Makefile
+
+# lib name, version
+LIBNAME=spdlog
+VERSION=1.5.0
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.gz
+
+CMAKE_OPTIONS= \
+	-DSPDLOG_BUILD_EXAMPLE=OFF \
+	-DSPDLOG_BUILD_TESTS=OFF \
+	-DSPDLOG_BUILD_BENCH=OFF \
+	-DSPDLOG_INSTALL=ON \
+	-DSPDLOG_FMT_EXTERNAL=ON
+
+ifeq ($(CROSS_COMPILING), yes)
+  DEPS += ../../Makefile.include
+else
+  CXXFLAGS += -std=c++11
```

Comment:
Why C++11?

Reply:
This is what I found in `crossguid` and it also seems to be used by `rapidjson`. Would you propose `c++14`? I couldn't find any depends build using that so far.

Reply:
Doesn't the library itself select the correct C++ version it needs?
If it doesn't, C++14 should be used as that's the one Kodi uses.

Reply:
No you manually have to specify whether your project support C++11 or C++14 and the library adapts depending on that. I can change it to C++14 it just doesn't fit what we use in other depends.

Reply:
I've changed this to `-std=c++14`.